Update README.md (#10)
peterbe/video-to-screenshots: README.md

> Demo app for how to create screenshots from a video file.

Author: ```suggestion
A web app to turn a video file into screenshots without a server.
```
